registry/src/model/mod.rs

```diff
@@ -0,0 +1,67 @@
+// Copyright 2023 Helsing GmbH
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#![allow(missing_docs)]
+
+use atmosphere::prelude::*;
```

Comment:
I personally dislike both star imports (they obfuscate what's actually used and where it's pulled from) and preludes.